src/multi-stage-output.tsx

```diff
@@ -294,7 +294,7 @@ class CIMultiStageOutput<T extends Record<string, unknown>> {
 
         const formattedData = info.get ? info.get(this.data as T) : undefined
         if (!formattedData) continue
-        const key = info.type === 'message' ? formattedData : info.label
+        const key = info.type === 'message' ? formattedData : `${info.label}: ${formattedData}`
```

Comment:
key will include dynamic data so new output based on updated data will always be printed

Reply:
The key is only the label for dynamic data because, otherwise, it will print on every single update - which is what we wanted to avoid

Run `OCLIF_CI_UPDATE_FREQUENCY_MS=500 CI=true tsx examples/sf-specific/deploy.ts` before and after your change to get an idea of the difference

Reply:
Just thinking out loud here: I wonder if our best option is to add a new option to `InfoBlock` that will tell the CI output to always print - regardless of it the update timeout has been met or not.

So something like this:

```
{
  label: 'Successful',
  get: (data): string | undefined => {
     // formatting stuff here
  }
},
{
  label: 'Failed',
  alwaysPrintInCI: true,
  get: (data): string | undefined => {
     // formatting stuff here
  }
}
```

That way customers always get updates `Failed` but only intermittently get updates to `Successful`

Reply:
> Run OCLIF_CI_UPDATE_FREQUENCY_MS=500 CI=true tsx examples/sf-specific/deploy.ts before and after your change to get an idea of the difference

ohh I see now.

> Just thinking out loud here: I wonder if our best option is to add a new option to InfoBlock that will tell the CI output to always print - regardless of it the update timeout has been met or not.

I like that option, I was struggling in PDR trying to find why updates were missing til I found about the throttle env affected it (which lead me to make this change first).
With the new option, the InfoBlock getters would need explicitly handle possible duplicates like this:
https://github.com/salesforcecli/plugin-deploy-retrieve/blob/fcc83af585eca9fc8f47db48a827a645e7fed116/src/utils/deployStages.ts#L161C1-L168C16

I'll add it in a bit 👍🏼